### PR TITLE
TM-1126 filter variables in conditions

### PR DESCRIPTION
**Pull request by MarcinMat**

### CODE REVIEW

File: src/SevenTag/Api/TriggerBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -5,7 +5,10 @@ imports:
 services:
     seven_tag.trigger_type.condition_preparator:
         class: SevenTag\Api\TriggerBundle\TriggerType\ViewPreparator\ConditionViewPreparator
-        arguments: [@seven_tag_variable.variable_manager]
+        arguments:
+                    - @seven_tag_variable.repository.variable_repository
```
Comment (kallosz) [inline src/SevenTag/Api/TriggerBundle/Resources/config/services.yml]: wrong indent

File: src/SevenTag/Api/TriggerBundle/TriggerType/ViewPreparator/ConditionViewPreparator.php [hunk 2]
```diff
@@ -91,4 +117,27 @@ public function getView(HolderInterface $holder, ContainerInterface $container)
...
+    {
+        $allowedVariables = $type->getAllowedVariables();
+        $definedVariablesForType = [];
+
+        /** @var Variable $variable */
+        foreach ($definedVariables as $key => $variable) {
+            $isAllowed = false;
+
+            /** @var GenericVariableType $allowedVariableType */
+            foreach ($allowedVariables as $allowedVariableType) {
```
Comment (kallosz) [inline src/SevenTag/Api/TriggerBundle/TriggerType/ViewPreparator/ConditionViewPreparator.php]: to high complexity.

File: src/SevenTag/Api/TriggerBundle/TriggerType/ViewPreparator/ConditionViewPreparator.php [hunk 3]
```diff
@@ -31,17 +34,32 @@
...
+    private $repository;
 
     /**
-     * @param VariableManager $manager
+     * @param EntityRepository $repository
+     * @param VariableProvider $variableProvider
+     * @param VariableTypeProvider $variableTypeProvider
      */
-    public function __construct(VariableManager $manager)
+    public function __construct(EntityRepository $repository, VariableProvider $variableProvider, VariableTypeProvider $variableTypeProvider)
```
Comment (kallosz) [inline src/SevenTag/Api/TriggerBundle/TriggerType/ViewPreparator/ConditionViewPreparator.php]: missing phpunit tests for this class

**Comment by marnits** [comment]
please change destination branch to `release/1.3.0`

**Pull request closed without merging**